### PR TITLE
Fix for issue #794 (bugfix)

### PR DESCRIPTION
**Pull request by grasdk**
* Switched from ExifReader to exifr and updated tests and test material to fix issue [#794](https://github.com/bpatrik/pigallery2/issues/794)
* Removed dependency to ExifReader
* Updated .gitignore
* Minor clarification added to CONTRIBUTING.md

### CODE REVIEW

**Comment by grasdk** [comment]
While doing this, I discovered a bug with png files having the "createDate" set to the file-date, even if they have metadata embedded. Exifr can read this date metadata from png files, so there are options to further utilize exifr for fixing issues.

I will look into this, but wanted to separate it from the fix of the issue.

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,11 @@ test.*
 *.sublime-project
 *.sublime-workspace
 .DS_Store
+test/folder-reset.js
```
Comment (bpatrik) [inline .gitignore:37]: Can you just use wildcard instead, like: test/*.js etc?

**Review by bpatrik — CHANGES_REQUESTED** [review]
Thank you! Nice contribution!


Can you please also remove any unused dependencies from the package.json (and also from the lock file)

**Comment by grasdk** [comment]
> Thank you! Nice contribution!
> 
> Can you please also remove any unused dependencies from the package.json (and also from the lock file)

Sure. I will do this asap. Hopefully tonight (it's morning here, now). Also the .gitignore

**Comment by bpatrik** [comment]
Thank you!

**Pull request merged**